src/engine.rs

```diff
@@ -81,6 +82,16 @@ pub fn print_crlf(stdout: &mut Stdout) -> Result<()> {
     Ok(())
 }
 
+pub fn clear_screen(stdout: &mut Stdout) -> Result<()> {
+    let (_, num_lines) = terminal::size()?;
+    for _ in 0..2 * num_lines {
+        stdout.queue(Print("\n"))?;
+    }
+    stdout.queue(MoveTo(0, 0))?;
+    stdout.flush()?;
```

Comment:
You might be able to run this crossterm command for a clear in one step: https://docs.rs/crossterm/0.19.0/crossterm/terminal/enum.ClearType.html

Reply:
`ClearType::All` only blanks the currently visible screen, so you cannot scroll back up to the previous output as with bash et al.

Reply:
Today I learned :)